docs/wallet/plugins/0-passkeys.mdx

```diff
@@ -257,6 +258,7 @@ const swapOwnerWithPasskeys: MetaTransaction = await smartAccount.createSwapOwne
     nodeUrl, 
     webauthPublicKey, // the x and y webAuthn publickey
     oldOwnerPublicKey, // the old owner to replace
+    { nodeRpcUrl: RPC_URL }
```

Comment:
Thank you for the PR. 

The fix in `createAddOwnerWithThresholdMetaTransactions` looks good. 

However, in `createSwapOwnerMetaTransactions`, the `nodeUrl` is already passed on as the first function parameter.